.github/workflows/connect-test.yml

```diff
@@ -64,318 +64,65 @@ jobs:
           name: build-artifact-connect-iframe
           path: packages/connect-iframe/build
 
-  api:
-    needs: [build]
-    if: github.repository == 'trezor/trezor-suite'
```

Comment:
i think you might be missing these (relatively newly added)
`  if: github.repository == 'trezor/trezor-suite'`